kieker-analysis/build.gradle

```diff
@@ -12,6 +12,7 @@ dependencies {
 	clover 'org.openclover:clover:4.2.0'
 
 	compile project(':kieker-common')
+	compile project(':kieker-monitoring')
```

Comment:
Should :analysis have dependencies towards :monitoring?
Is there something that should go to :common instead?

Reply:
We have one sink filter which uses Kieker writers to serialize records. I could move it to tools, but this would be confusing to the user.